### DISCUSSION TITLE
How to set babel config options for internal jsx compiler (throwIfNamespace)

### DISCUSSION THREAD

**Discussion by adb-sh** [denoland/deno#18575] [Q&A]
Hey, I want to compile xml in deno with lume, not plain html. So I also need to compile xml namespaces in jsx.
Im using the following compiler opions:
```json
"compilerOptions": {
  "jsx":"react-jsx",
  "jsxImportSource": "npm:preact"
},
```
I get the following error message, when trying to compile:
```
Caused by TypeError: JSX Namespace is disabled by default because react does not support it yet. You can specify jsc.transform.react.throwIfNamespace to false to override default behavior
```
But unfortunately I didn't figured how to set the throwIfNamespace option. :/

**Comment by jollytoad** [comment]
This appears to be an option in [swc](https://swc.rs/docs/configuration/compilation#jsctransformreactthrowifnamespace), I think namespaces should be allowed by default, as this behaviour is also inconsistent with TypeScript, which is perfectly happy with the syntax.

So I've raised an issue... https://github.com/denoland/deno/issues/20345
